department-of-veterans-affairs/vets-website
# Add confirmation page

## Summary

- Implement the ConfirmationView component into the 21P-530 (Burials) form and update the confirmation page with a few MVP changes to meet the requirements of ZSF stage 2.

This is behind a flipper, which needs to be merged first https://github.com/department-of-veterans-affairs/vets-api/pull/19761. 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/95585
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/98098

## Testing done

- Disable http://localhost:3000/flipper/features/burial_confirmation_page
     - Fill out and submit http://localhost:3001/burials-memorials/veterans-burial-allowance/apply-for-allowance-form-21p-530ez/introduction
     - Ensure the confirmation page has no changes from `main`

- Enable http://localhost:3000/flipper/features/burial_confirmation_page
     - Fill out and submit http://localhost:3001/burials-memorials/veterans-burial-allowance/apply-for-allowance-form-21p-530ez/introduction
     - Ensure the confirmation page matches the [wireframes](https://www.figma.com/design/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?node-id=3800-5635&t=Pz6zX8tqzdOEUGe9-4)

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Authenticated  |   <img width="490" alt="Conf1a" src="https://github.com/user-attachments/assets/26afe7b9-0c6b-4adf-8bb2-d21ac8cd4e88"> <img width="463" alt="Conf1b" src="https://github.com/user-attachments/assets/741955d9-0e32-443b-aebf-246b1ce2b39e"> <img width="533" alt="Conf1c" src="https://github.com/user-attachments/assets/c1a36523-9e92-46d6-a7cb-2f0d0cfc8980">  |  <img width="472" alt="Conf2a" src="https://github.com/user-attachments/assets/981143f2-6850-4e24-9f05-7de24ccf914c"> <img width="349" alt="Screenshot 2024-12-09 at 10 31 49 AM" src="https://github.com/user-attachments/assets/570e0603-d210-4094-8716-0a9f40f27b3e"> <img width="484" alt="Conf2c" src="https://github.com/user-attachments/assets/3b7f87ba-4515-4fb9-9c29-ee046186d81f"> |
| Unauthenticated |    Duplicate    |     <img width="492" alt="Screenshot 2024-12-09 at 10 27 29 AM" src="https://github.com/user-attachments/assets/bc353ee3-92ae-467e-b4b5-25cb24220d18"> <img width="519" alt="Screenshot 2024-12-09 at 10 27 45 AM" src="https://github.com/user-attachments/assets/c649ab40-e451-4e52-88d0-686f97e62011">  |

## What areas of the site does it impact?

Burials

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
